### PR TITLE
ZEN-33033 Add endpoints for AWS modeler server

### PR DESCRIPTION
**Pull request by DeKapito**

### CODE REVIEW

File: services/Zenoss.cse/Zenoss/Collection/localhost/localhost/zenmodeler/service.json [hunk 1]
```diff
@@ -55,6 +55,13 @@
             "PortNumber": 8444,
             "Protocol": "tcp",
             "Purpose": "import"
+        },
+        {
+            "Application": "{{(parent .).Name}}_aws_modeler_server",
```
Comment (jpeacock-zenoss) [inline services/Zenoss.cse/Zenoss/Collection/localhost/localhost/zenmodeler/service.json:60]: What service exports this endpoint?

**Review by jpeacock-zenoss — COMMENTED** [review]
~There should be a migration script written to add this endpoint to the services.~
Premature question.  Where does this `localhost_aws_modeler_server` endpoint come from?

File: services/Zenoss.cse/Zenoss/Collection/localhost/localhost/zenmodeler/service.json (hunk 1 above)
Comment (DeKapito) [reply services/Zenoss.cse/Zenoss/Collection/localhost/localhost/zenmodeler/service.json:60]: zenpython

Comment (DeKapito) [reply services/Zenoss.cse/Zenoss/Collection/localhost/localhost/zenmodeler/service.json:60]: This pr also adds the export for zenpython service

**Review by jpeacock-zenoss — CHANGES_REQUESTED** [review]
Please include a service migration script to add this endpoint to the service definitions.

**Comment by romanbakaleyko** [comment]
service migration script is in the zenpack https://github.com/zenoss/ZenPacks.zenoss.AWS/blob/d40f7fff0d97ff7d25f7d8d7d015cbd72d185acc/ZenPacks/zenoss/AWS/service_migration.py

**Review by jpeacock-zenoss — APPROVED** [review]

**Comment by jpeacock-zenoss** [comment]
OK.  I see that the service definitions are updated when the AWS ZenPack is installed.  Since ZenPacks are re-installed during an upgrade, then this should take care of the migration.

**Pull request merged**